adapters/tlsio_schannel.c

```diff
@@ -746,7 +746,7 @@ static void on_underlying_io_bytes_received(void* context, const unsigned char*
                     {
                         LPVOID srcText = NULL;
                         if (FormatMessage(FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS, NULL,
-                            status, MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT), (LPTSTR)srcText, 0, NULL) > 0)
+                            status, MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT), (LPTSTR)&srcText, 0, NULL) > 0)
```

Comment:
this changes parameter from srcText to &srcText, by design?

Reply:
Yes, this is a bug fix; with FORMAT_MESSAGE_ALLOCATE_BUFFER this parameters is a return parameter, returning the newly allocated buffer.